src/istio/zarf.yaml

```diff
@@ -86,7 +88,7 @@ components:
     required: true
     import:
       path: common
-
+  # @lulaStart 0a4b40fa-7d87-4523-a64d-304969a47b44
```

Comment:
On these we may want to just wrap the gateway name or the values files? Otherwise they will trigger on each Istio version bump.